src/EmployerCohortApproved.txt

```diff
@@ -0,0 +1,16 @@
+Name: EmployerCohortApproved
+Dev TemplateId: 70e3bc3e-03ed-433f-8729-059c1377d477
+Prod TemplateId: 2e4ea7ce-b4fb-44f6-b74a-c29e67a68810
+
+From: The Apprenticeship Service
+To: email address
+Subject: Apprentice details ready to approve
+Body:
+((provider_name)) sent you apprentice details to approve.
+Sign into your apprenticeship service account to view these details � https://manage-apprenticeships.service.gov.uk/commitments/accounts/((employer_hashed_account))/apprentices/((cohort_reference))/details
```

Comment:
I'm not sure what part this file plays in the generation of emails (I thought the templates were stored in some central gateway somewhere)... the two characters that look a hyphen and an apostrophe are the characters from the extended character set - will it make it through to the emails?

Reply:
They are, but Chris asked if we could record the templates here (once we make any changes to them). It's just so they are centralised